app/commands/v2/publish.rb

```diff
@@ -163,6 +167,40 @@ def set_public_updated_at
         end
       end
 
+      def set_temporary_major_published_at
+        return unless update_type == "major"
+
+        edition.update_attributes!(
+          temporary_major_published_at: default_datetime
+        )
+      end
+
+      def set_temporary_minor_published_at
+        return unless update_type == "minor"
+
+        edition.update_attributes!(
+          temporary_minor_published_at: default_datetime
+        )
+      end
+
+      def set_temporary_last_edited_at
+        if update_type.present?
```

Comment:
I don't think we need this, if the `update_type` isn't present then the publish will fail anyway.